run/models.py

```diff
@@ -0,0 +1,43 @@
+from django.db import models
+from storage.models import File
+from game_runner import settings
+import uuid
+
+
+# Initial scheme
+# FilePath
+#   +file : File
+#   +path : String
+#   +run : Run
+#
+# Run
+#   +token : Token
+#   +request_time : DateTime
+#   +start_time : DateTime
+#   +end_time : DateTime
+#   +input_file_paths : FilePath[]
+#   +output_file_paths : FilePath[]
+#   +log : File
+#   +game : Game
+#
+# API Methods:
+#   +run((game : Game, file_paths : FilePath[])[]) : tokens String[]
+#   +report(from_time : DateTime) : runs : Run[]
+
+class FilePath(models.Model):
+    file = File()
+    path = models.FilePathField()
```

Comment:
I think they are ForeignKey to storage.models.File
And as I remember there was `client_1`, `client_2`, `server` files.
@akmohtashami knows better :smiley: